d3a_api_client/__init__.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, area_id, *args, **kwargs):
         pass
 
     @abstractmethod
-    def register(self, is_blocking):
+    def register(self):
```

Comment:
Why did you opt for removing the parameter here? Wouldn't it be better to keep the same signature for the method across interface and implementations?

Reply:
You are right, remnant of refactorization. Changing it back